src/contracts/middleware/IndexRegistry.sol

```diff
@@ -225,7 +226,7 @@ contract IndexRegistry is IIndexRegistry {
             uint256 listIndex = (totalOperatorsHistoryLength - 1) - i;
             OperatorIndexUpdate memory totalOperatorUpdate = _totalOperatorsHistory[quorumNumber][listIndex];
             // look for the first update that began at or after `blockNumber`
-            if (totalOperatorUpdate.fromBlockNumber >= blockNumber) {
+            if (totalOperatorUpdate.fromBlockNumber <= blockNumber) {
```

Comment:
is this a reversion? seems like this change is purely making the behavior wrong?

Reply:
it's looping backwards now, will fix the comment

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/88/commits/8888f2a31ff9e13bca8370846c35f1c48b45d48f

Reply:
agreed this is correct now, was confused myself, thanks.